docs/config.ini

```diff
@@ -20,9 +20,9 @@ release = 0.0.15
 # items: name, URL, and a fontawesome icon
 # (https://fontawesome.com/icons?d=gallery). Items are separated by commas.
 header_links = Stable Version Documentation, https://autogluon.mxnet.io/stable/index.html, fas fa-book,
```

Comment:
Ideally we will also want a More Versions button that has a hardcoded link to 
```http://autogluon.mxnet.io/dev/versions.html```
and the page contains hardcoded links to the other known documentation versions such as 0.0.14, dev, stable, 0.x.y, etc.

Reply:
We will have to backport this (and other changes) to 0.0.14/stable so they can link properly, then things should work properly going forward.

Reply:
Fixed the links, now they are branch-specific